src/metabase/task/sweep_query_analysis.clj

```diff
@@ -0,0 +1,90 @@
+(ns metabase.task.sweep-query-analysis
+  (:require
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.cron :as cron]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [metabase.query-analysis :as query-analysis]
+   [metabase.task :as task]
+   [toucan2.core :as t2])
+  (:import
+   (org.quartz DisallowConcurrentExecution)))
+
+(set! *warn-on-reflection* true)
+
+(defn- analyze-cards-without-query-fields!
+  ([]
+   (analyze-cards-without-query-fields! query-analysis/analyze-sync!))
+  ([analyze-fn]
+   (let [cards (t2/reducible-select [:model/Card :id]
+                                    {:left-join [[:query_field :f] [:= :f.card_id :report_card.id]]
```

Comment:
nitpick, but I've been religiously using `qf` as the alias since more often than not `Field` gets joined in too and that would be `f`.